general/documentation/installation.md

```diff
@@ -58,19 +58,23 @@ Gitpod is an alternative to local development and completely optional. We recomm
 
 ## Useful commands
 
-### Starting the development server
+### Starting the development server - the best way to edit docs locally
 
 ```console
 yarn start
 ```
 
+Once this completes, the docs will open in your browser. You can just edit the .md source files and reload to see the effect of your edits.
```

Comment:
The files shoudl live reload?

Reply:
That seems like Jargon to me. I prefer the plain English, which I jsut edited a bit.